swarm/api/http/server.go

```diff
@@ -364,16 +364,44 @@ func (s *Server) HandleDelete(w http.ResponseWriter, r *Request) {
 	fmt.Fprint(w, newKey)
 }
 
+// possible combinations:
+// /			add multihash update to existing hash
+// /raw 		add raw update to existing hash
+// /#			create new resource with first update as mulitihash
+// /raw/#		create new resource with first update raw
+func resourcePostMode(path string) (isRaw bool, frequency uint64, err error) {
+	fields := strings.Split(path, "/")
+	freqUrlIdx := -1
+	if fields[0] != "" {
```

Comment:
why not use a regular expression here to do the matching?

Reply:
Thank you! %)